cmd/admin-router.go

```diff
@@ -260,7 +260,7 @@ func registerAdminRouter(router *mux.Router, enableConfigOps bool) {
 
 		// -- KMS APIs --
 		//
-		adminRouter.Methods(http.MethodPost).Path(adminVersion + "/kms/status").HandlerFunc(gz(httpTraceAll(adminAPI.KMSStatusHandler)))
+		adminRouter.Methods(http.MethodGet).Path(adminVersion + "/kms/status").HandlerFunc(gz(httpTraceAll(adminAPI.KMSStatusHandler)))
```

Comment:
you don't have to change the existing APIs you should be only concerned with kms-router.go @reivaj05